CODEOWNERS

```diff
@@ -221,6 +221,7 @@
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
 * @cilium/tophat
+/.github/workflows/*perf*.yaml @cilium/sig-scalability @cilium/github-sec @cilium/ci-structure
```

Comment:
I don't think we need to block the PR on this but I wanted to point out to @marseel / @cilium/sig-scalability that in general we expect most groups to apply their review feedback during the primary PR against `main` then the backport is a sanity check, so for this case @cilium/sig-scalability does not need to feel obliged to review all changes to stable branch modifications of this workflow. Perhaps as a followup we can just drop this change and defer to the existing rule.

```suggestion
```

Reply:
Yes, that is totally fine with me - modification of CODEOWNERS was part of the commit where I introduced a new test, which seems reasonable in general, but I guess for backports we should get rid of it. I will open followup PR.